packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -96,7 +100,7 @@ const objectToComponentProps = (obj, options = { exclude: [] }) => {
 // returns the content of App.{js,tsx} with <AuthProvider> added
 const addWebRender = (content, authProvider) => {
   const [_, indent, redwoodApolloProvider] = content.match(
-    /(\s+)(<RedwoodApolloProvider>.*<\/RedwoodApolloProvider>)/s
+    /(\s+)(<RedwoodApolloProvider.*>.*<\/RedwoodApolloProvider>)/s
```

Comment:
Instead of looking for `<RedwoodApolloProvider` here, and wrapping `<AuthProvider` around that, I think it would be better to look for `<RedwoodProvider` and make `<AuthProvider` wrap all its children.

The reason for this is we support swapping out the Apollo provider, so it's not guaranteed to exist in all RW projects.

Reply:
ok - will need to write some new regex for this